src/widgets/call-panel.jsx

```diff
@@ -0,0 +1,396 @@
+    this.forceUpdate();
+  }
+
+  handleToggleCameraClick() {
+    this.toggleMedia('video');
+  }
+
+  handleToggleMicClick() {
+    this.toggleMedia('audio');
+  }
+
+  render() {
+    if (!this.props.topic) {
+      return null;
+    }
+
+    const remoteActive = this.remoteRef.current != null && this.remoteRef.current.srcObject != null;
+    const audioIcon = this.state.localStream != null && this.state.localStream.getAudioTracks()[0].enabled ? 'mic' : 'mic_off';
+    const videoIcon = this.state.localStream != null && this.state.localStream.getVideoTracks()[0].enabled ? 'videocam' : 'videocam_off';
+
+    return (
+      <>
+        <div id="topic-caption-panel" className="caption-panel">
+          {this.props.displayMobile ?
+            <a href="#" id="hide-message-view" onClick={(e) => {e.preventDefault(); this.props.onHideMessagesView();}}>
+              <i className="material-icons">arrow_back</i>
+            </a>
+            :
+            null}
+          <div className="avatar-box">
+            <LetterTile
+              tinode={this.props.tinode}
+              avatar={this.props.avatar}
+              topic={this.props.topic}
+              title={this.props.title} />
+          </div>
+          <div id="topic-title-group">
+            <div id="topic-title" className="panel-title">{
+              this.props.title ||
+              <i><FormattedMessage id="unnamed_topic" defaultMessage="Unnamed"
+                description="Title shown when the topic has no name" /></i>
+            }</div>
+          </div>
+        </div>
+        <div id="video-container">
+          <div className="video-container-panel">
+            <div className="video-elem">
+              <video id="localVideo" ref={this.localRef} autoPlay muted playsInline></video>
+              <div className="video-title">
+                <FormattedMessage id="calls_you_label"
+                  defaultMessage="You" description="Shown over the local video screen" />
+              </div>
+            </div>
+            <div className="video-elem">
+              <video id="remoteVideo" ref={this.remoteRef} autoPlay playsInline></video>
+              {remoteActive ?
+                <div className="video-title">{clipStr(this.props.title, MAX_TITLE_LENGTH)}</div>
+                :
+                null}
+            </div>
+          </div>
+          <div id="video-container-controls">
+            <button className="video-call-hangup" onClick={this.handleCloseClick}>
+              <i className="material-icons">call_end</i>
+            </button>
+            <button className="video-call-toggle-media" onClick={this.handleToggleCameraClick}><i className="material-icons">{videoIcon}</i></button>
+            <button className="video-call-toggle-media" onClick={this.handleToggleMicClick}><i className="material-icons">{audioIcon}</i></button>
+          </div>
+          <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
```

Comment:
This should be included as a dependency in package.json instead of being loaded directly here.